examples/nlp/fnet_classification_with_keras_nlp.py

```diff
@@ -160,19 +158,11 @@
 
 
 def train_word_piece(ds, vocab_size, reserved_tokens):
```

Comment:
why not just remove this whole function and just inline?

also, above this in the paragraph replace

"Luckily, TensorFlow Text
makes it very simple to train WordPiece on a corpus as described in
[this guide](https://www.tensorflow.org/text/guide/subwords_tokenizer)."

with

"We can train a WordPiece vocabulary ourselves with `keras_nlp.tokenizers.compute_word_piece_vocabulary`."